### PR TITLE
Verbosity and error handling improvements in bulk_register

### PR DESCRIPTION
**Pull request by MTLehner**
Multiple small improvements to the verbosity and error prints in the function bulk_register

1. A progress bar for bulk_register
    - A lightweight `tqdm` bar function is added
    - The progress bar is by default turned off and can be activated for long jobs, to get some ETA estimations
2. Changes to function arguments for consistent snake case naming
3. Catch kekulize error in MolToV3KMolBlock

### CODE REVIEW

**Comment by greglandrum** [comment]
I'm not a fan of progress bars, but if we're going to add one it seems like it makes sense to use `tqdm`, instead of rolling our own. tqdm is a lightweight dependency, wouldn't it make sense to just use that?

File: lwreg/utils.py [hunk 1]
```diff
@@ -843,6 +851,15 @@ def bulk_register(config=None,
                 res.append((mrn, conf_id))
         except _violations:
             res.append(RegistrationFailureReasons.DUPLICATE)
+        except psycopg2.errors.ProgramLimitExceeded as e:
+            # check if the exception is ProgramLimitExceeded and config has a
+            # 'registerConformers'==1, if so print user friendly message
+            if _lookupWithDefault(config, "registerConformers"):
+                raise Exception(f"Error registering molecule with index: {mol_idx}.\nProblem might be due to the length of the conformer hash.\nTry reducing the 'numConformerDigits' or the number of atoms in the molecule") from e
```
Comment (greglandrum) [inline lwreg/utils.py:858]: Asking the user to reduce the number of atoms in the molecule isn't exactly helpful ;-)
Maybe drop that part of the message.

**Comment by greglandrum** [comment]
Though doing an informative error message certainly makes sense, it feels like it would be good to find an actual solution to the conformer hashing problem. There's really no reason for us to be storing the whole string of coordinates as a hash (this was just me not thinking through the consequences completely), we could just generate the SHA256 from the string and store that instead. 

It would help keep the database size down too, and that's not a terrible thing.

It would be a separate PR and would require having some code to transition existing lwreg installs, but that's not a big deal.

We can quickly talk about it later this week with @brje01 and I'll do that PR.

**Comment by MTLehner** [comment]
> I'm not a fan of progress bars, but if we're going to add one it seems like it makes sense to use `tqdm`, instead of rolling our own. tqdm is a lightweight dependency, wouldn't it make sense to just use that?

I did it in my own fork with `tqdm` and it worked great, but adds the additional dependency on `tqdm`. Which is pretty light, and easy to use. Discussion with @brje01 lead us to the even lighter function, taken from here: [https://stackoverflow.com/questions/3160699/python-progress-bar](https://stackoverflow.com/questions/3160699/python-progress-bar)
Which looks and works similar to `tqdm`, but without all the extra options.

**Comment by greglandrum** [comment]
As we discussed, I agree that making the argument naming scheme consistent is a good idea (even though it's a bit of a disrupitive change). The tutorial and demo notebooks also need to be updated to reflect that change.

**Pull request merged**